bigquery_etl/events_daily/query_templates/event_types_v1/templating.yaml

```diff
@@ -14,3 +14,7 @@ firefox_accounts_derived:
   name: Firefox Accounts
   dataset: firefox_accounts
   dag_name: bqetl_event_rollup
+mozilla_vpn_derived:
+  name: Mozilla VPN
+  dataset: mozilla_vpn
+  dag_name: bqetl_event_rollup
```

Comment:
I wound up just adding this to the default dag we use for Firefox, I didn't feel it was worth creating a new one just for vpn. lmk if you disagree